tests/test_one_client_server.py

```diff
@@ -83,20 +83,14 @@ async def connect_callback(self, server: tcpip.OneClientServer) -> None:
             )
         await super().connect_callback(server)
 
-    def sync_connect_callback(self, server: tcpip.OneClientServer) -> None:
-        """A synchronous version of connect_callback.
-
-        TODO DM-37477: drop this method argument when we remove support
-        for synchronous connect_callback functions.
-        """
-        print(f"sync_connect_callback: connected={server.connected}")
+    def sync_connect_callback(self) -> None:
```

Comment:
Shouldn't this be removed?

Reply:
No, it shouldn't be removed. The purpose of this synchronous callback function is to support the unit test that shows that synchronous callback functions are not supported anymore.

Reply:
I added that sentence to the doc string of the function.